src/frontend/src/lib/styles/global/colors.scss

```diff
@@ -16,6 +16,7 @@
 	--color-medium-purple: theme(colors.medium-purple);
 	--color-platinum: theme(colors.platinum);
 	--color-mountain-meadow: theme(colors.mountain-meadow);
+	--color-dodger-blue: #016dfc;
```

Comment:
Can you configure the new color as a tailwind variable as well?

Reply:
done!